_posts/2022-04-27-newpipe-0.23.0.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "NewPipe 0.23.0 released: Behold the behemoth"
+short: "NewPipe 0.23.0 released"
+date: 2022-04-27 8:00:00 + 01:00
+categories: [pinned, release]
+author: <a href="https://github.com/opusforlife2">@Opusforlife</a> & <a href="https://github.com/poolitzer">@Poolitzer</a>
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+This release got delayed by quite a bit, and so it just grew and grew and grew. You're getting a truly massive release to make up for the long wait. Be ready for a whopper of a blog post!
+
+<!-- more -->
+
+Wow. Where do we even start? Let's see... new stream notifications! Now your phone can ping you even more frequently than before! Now you get seamless transition to and from background! Now there is no pause in the audio when switching between apps! YouTube/YouTube Music Mixes, the app suggests you these in the related items so that you can binge curated content to your hearts'... content. And finally, the popup player is usable again on Android 12!
+
+This post also contains all the changes from the 0.22.0 Extractor version, which we were forced to release early as part of the hotfix to fix YouTube playback breaking yet again.
+
+## New
+
+- [@nv95](https://github.com/nv95) brings you a highly requested and anticipated feature: Notifications for new streams from subscribed channels! Now whenever your favourite YouTuber uploads a new video, NewPipe can send you a notification to let you know!
+
+
+- [@TiA4f8R](https://github.com/TiA4f8R) brings you a second frequently requested and really awesome feature. Seamless switch to and from background! Now when you select "Minimize to background" in settings and minimise the app, NewPipe will pause the video stream and continue the audio stream without interruption! When you switch back to the app, you will see a black screen for a couple of seconds, after which the video will resume playing. (Note: there are certain (usually very old) videos for which this won't work. In those cases, the regular short pause in the audio will occur before playback resumes.)
+
+- [@Stypox](https://github.com/Stypox) made it so that YouTube Mixes and YouTube Music Mixes show up in related videos section. You're no longer forced to do the tedious job of having to share Mix URLs to NewPipe each time from another app to be able to play them! For those who have discovered upon reading this that NewPipe can play Mixes, congrats, and have fun! :)
```

Comment:
```suggestion
- [@Stypox](https://github.com/Stypox) ~~made it so that YouTube Mixes and YouTube Music Mixes show up in related videos section. You're no longer forced to do the tedious job of having to share Mix URLs to NewPipe each time from another app to be able to play them! For those who have discovered upon reading this that NewPipe can play Mixes, congrats, and have fun! :)~~ Well, actually YouTube already broke this feature by forcing users to accept cookies in order to show mixes in related items...
```